src/parse.d

```diff
@@ -4087,10 +4087,16 @@ public:
                     pAttrs.storageClass &= (STCsafe | STCsystem | STCtrusted);
                 }
                 Dsymbol s = v;
+                if (structalign != 0)
```

Comment:
Oh, this is a third meaning given to the alignment value (``>0``, ``~0`` and ``0``)